torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -176,6 +176,16 @@ bool needsProfiledInputs(Node* n) {
   }
 }
 
+bool inputsProfilable(Node* n) {
+  for (auto iv : n->inputs()) {
+    if (!needsProfiledInputs(iv->node()) && n->kind() != prim::Constant) {
```

Comment:
constants don't have any inputs so you shouldn't need to add the `&& n->kind() != prim::Constant` case. or maybe you meanat `iv->node()` ?

Reply:
yes, this is wrong. We should just skip over constants. Let me fix this